### DISCUSSION TITLE
opened PR statistics

### DISCUSSION THREAD

**Discussion by zeol** [renovatebot/renovate#28271] [Suggest an Idea]
### Tell us more.

Hello,
I can see Renovate already provides a lot of statistics during it's run.
Would it be feasible to include an information regarding current number of PRs opened by Renovate (waiting for a merge) ?
I could actually use such information to impose some pression on DEV teams :)

**Comment by secustor** [comment]
This can be derived from OpenTelemetry traces, logging message in debug mode ( which is easier in JSON log mode ) or the report feature.

**Reply by zeol** [reply]
In theory yes, however my current self-hosted (helmchart) setup inspects around 700 repos. I use JSON format and send logs to NewRelic. Using DEBUG mode produces hundred thousands of logs so I'd rather avoid it.  There's a lot stats.manager.* metrics already present in logs and I hoped that also others could benefit from the new one I've mentioned.

**Reply by viceice** [reply]
you can log level remap to map your required debug messages to info level 

https://docs.renovatebot.com/configuration-options/#loglevelremap

**Reply by zeol** [reply]
Awesome suggestion, I've missed this option when reading docs. Thanks!

**Reply by viceice** [reply]
it's relatively new
